docs/extensions/model-merger.md

```diff
@@ -0,0 +1,66 @@
+# Model Merger
+
+The model merge is used to automatically merge two models. Merge is not meant in the same way as in terms VCS systems like git. It‘s main use case is incrementalisation of importer at the moment but is not limited to it.
+## Model Merge Specification 
```

Comment:
Is it intentional that above it says "Model Merger"   and here "Model Merge"?

Reply:
I thought "merger" specification sounds strange that's why I chose "merge"